hathor/event/storage/memory_storage.py

```diff
@@ -33,3 +33,10 @@ def get_event(self, key: int) -> Optional[BaseEvent]:
             return None
 
         return self.events[key]
+
+    def get_last_event(self) -> Optional[BaseEvent]:
+        if not self.events:
+            return None
+
+        last_id = list(self.events)[-1]
```

Comment:
What's the type of `self.events`? I mean, the keys are integers but what are their meaning? Index? Why not a list instead?

Reply:
Done.

Reply:
Great! Thanks!